### DISCUSSION TITLE
[Performance] Opening object with Master layout

### DISCUSSION THREAD

**Discussion by Maksonis** [pimcore/pimcore#12456] [Q&A / Support]
I have an issue with opening object with Master Layout. Response json is around 70MB:

![image](https://user-images.githubusercontent.com/17277953/174753364-4524ba31-5b5f-4a52-a390-48b0ca41eb44.png)

but with Master (Admin) is around 1MB (same product):

![image](https://user-images.githubusercontent.com/17277953/174753855-cebc2014-1df9-438c-ba1f-60a45ef846a0.png)

![image](https://user-images.githubusercontent.com/17277953/174753759-21bcf134-ac76-4e9f-8420-3daca0e7ff36.png)

I think it's because childs that is shown only in Master layout here:

![image](https://user-images.githubusercontent.com/17277953/174754180-8d7a231b-0f92-4915-b0c3-943ce1d1b073.png)

and makes everything doubled (or more).

And childs doesn't exists in Master (Admin) layout:

![image](https://user-images.githubusercontent.com/17277953/174754358-5a6d4787-cf1e-4be9-8f33-68197435d1cf.png)

Is that a bug or ?

**Comment by dvesh3** [comment]
@Maksonis this is a BC layer for Childs(deprecated) property introduced here https://github.com/pimcore/pimcore/pull/11762 so duplicate properties are there with different indices as it could be used in third party bundles and removing it leads toa  breaking change.

I wonder why the response is so big in your case 🤔 

Could you please share your class definition?

**Reply by Maksonis** [reply]
@dvesh3 thank you for quick reply. My class definition contains 3 classification store attributes, which in total has 1,5k keys (with around 150 calculated ones) and around 50 fields (input, images, relations) . Not all CS attributes are maintained of course, and I use Hide Empty Data checkbox:

![image](https://user-images.githubusercontent.com/17277953/175011933-db40cbdd-88ea-4cfb-bf79-23e486e79cd1.png)

Without Classification Store attributes, response is 2MB (which is fine). So product has lot of cs attributes.
